crossandeancommands/make_nexus.py

```diff
@@ -0,0 +1,81 @@
+"""
+Create a nexus file from the most recent data on Edictor.
+"""
+from lingpy import Wordlist
+from lingpy.convert.strings import write_nexus
+from pathlib import Path
+
+
+def run(args):
+
+    wl = Wordlist.from_cldf(
+            Path("./", "cldf", "cldf-metadata.json").as_posix(),
```

Comment:
Please do not use relative paths, the CLDF commands work from everywhere in your computer. So you should instead use:

```python
from lexibank_crossandean import Dataset as CA
```

then in `def run`:

```python
    ds = CA()
    wl = Wordlist.from_cldf(str(ds.cldf_dir.joinpath("cldf-metadata.json")) # etc
```